packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -365,6 +365,9 @@ class InkResponse extends StatelessWidget {
   final GestureLongPressCallback? onLongPress;
 
   /// Called when the user taps this part of the material with a secondary button.
+  ///
+  /// The secondary button is typically accessed using the right mouse
+  /// button on desktop platforms such as Windows, macOS, and Linux.
```

Comment:
```suggestion
  /// The secondary button is typically accessed using the right mouse
  /// button on desktop platforms such as Windows, macOS, and Linux.
```

If it's a mouse button, it does't necessarily only for Windows, macOS, and Linux. You can connect mouse to other devices.  

According to some Google search, the secondary button is the right mouse button for right-handed users and left mouse button for the left-handed users.

Reply:
Thanks for suggestion! I'm sending changes